### DISCUSSION TITLE
Add new device to docs (install procedure)

### DISCUSSION THREAD

**Discussion by J1Buskens** [arendst/Tasmota#19829] [Q&A] [answered]
I was able to get Tasmota running on an unsupported device (DoHome plug). 
the install procedure is not as described in existing docs as it is using DoHome's own firmware uploader and some intermediary steps.
Is there a way I can add this information to the documentation for other people to follow?
The only option I seem to find is adding new templates. (It doesn't seem to be template related, it just worked directly after installing)

**Comment by sfromis** [comment, marked as answer]
The templates repository is a good place to describe the steps to getting Tasmota flashed to a particular device, and of course also the template config needed. Many of the template entries do have quite a bit of text when a device may be tricky to flash, or special considerations to make it workable. This extends well beyond just the pins.
